### PR TITLE
104 overriding mui styles without important

### PR DESCRIPTION
**Pull request by oonasaksman**
## Describe your changes

Removed all !importants. Added hover and focus classes to overwrite MUI for those cases.

## Checklist before requesting a review

-   [x] I have performed a self-review of my code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added thorough tests

### CODE REVIEW

**Review by Marakke — APPROVED** [review]
I cannot check the UI anymore, but if everything looks correct, then all good! Codewise looks fine at least :)

**Review by vrindavj — APPROVED** [review]
Good job!!

**Review by Marakke — DISMISSED** [review]
Nice work!

**Review by vrindavj — DISMISSED** [review]
Looks good

File: frontend/package-lock.json [hunk 1]
```diff
@@ -19864,6 +19864,8 @@
             "resolved": "https://registry.npmjs.org/@csstools/selector-specificity/-/selector-specificity-2.0.2.tgz",
             "integrity": "sha512-IkpVW/ehM1hWKln4fCA3NzJU8KwD+kIOvPZA4cqxoJHtE21CCzjyp+Kxbu0i5I4tBNOlXPL9mjwnWlL0VEG4Fg==",
             "requires": {}
+<<<<<<< HEAD
+=======
```
Comment (Jontzii) [inline frontend/package-lock.json:19868]: Remove these remains of git conflict

**Review by Marakke — APPROVED** [review]

**Review by vrindavj — APPROVED** [review]
approved

**Pull request merged**